vignettes/reproducing_pct_results.Rmd

```diff
@@ -0,0 +1,255 @@
+---
+title: "An Introduction to the pct package 2/2"
+subtitle: "Reproducing the results of the PCT"
+author: 
+- "Robin Lovelace and Layik Hama"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Reproducing the results of the PCT}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: refs.bib
+csl: nature.csl
+---
+
+```{r, eval=FALSE, echo=FALSE}
+# Aim: generate references.bib - run only if references change
+refs = RefManageR::ReadZotero(group = "418217", .params = list(collection = "8Y9DU4DR", limit = 100))
+RefManageR::WriteBib(refs, "vignettes/references.bib")
+citr::tidy_bib_file(
+  rmd_file = "vignettes/reproducing_pct_results.Rmd",
+  messy_bibliography = "vignettes/references.bib",
+  file = "vignettes/refs.bib")
+file.remove("vignettes/references.bib")
+```
+
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Comparing downloaded data with the PCT web app
+
+To compare the downloaded data with data in the PCT web app, we will take a subset of the `wight_lines_pct` dataset.
+The top 30 travelled desire lines by number of commuters who use cycling as their main mode is taken in the following code chunk.
+The reason for selecting the top 30 will become apparent (the `wight_lines_30` object is provided in the PCT package):
+
+```{r, eval=FALSE}
+line_order = order(wight_lines_pct$bicycle, decreasing = TRUE)
+wight_lines_30 = wight_lines_pct[line_order[1:30], ]
+```
+
+The resulting `wight_lines_pct` and `wight_lines_30` datasets are available in the package.
+We'll use the smaller one for speed.
+Note: these contain many variables, three of which (the number of people cycling, driving and walking along the desire lines from the 2011 Census) are shown below for the Isle of Wight:
+
+```{r, pct-lines-min}
+lwd = wight_lines_30$all / mean(wight_lines_30$all) * 5
```

Comment:
The problem here is that the object `wight_lines_30` has not yet been created. I suggest starting from a more recent version of the package (I made a load of changes to the `pct.Rmd` file in the last few days) may be the best way forward.